CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
+## 0.4.7
```

Comment:
That seems better. It wasn't clear from the previous releases when we bumped what.